app/templates/components/footer-main.hbs

```diff
@@ -43,10 +43,25 @@
 
     <div class="three wide column">
       <div class="ui inverted link list">
-        <strong class="item">{{t 'Follow us on'}}</strong>
-        <a class="item" href="#" target="_blank" rel="noopener noreferrer"><i class="facebook f icon"></i> Facebook</a>
-        <a class="item" href="#" target="_blank" rel="noopener noreferrer"><i class="twitter icon"></i> Twitter</a>
-        <a class="item" href="#" target="_blank" rel="noopener noreferrer"><i class="github icon"></i> Github</a>
+        <strong class="item">{{t 'Connect with us'}}</strong>
+        <a class="item" href="{{if socialLinks.supportUrl socialLinks.supportUrl '#'}}" target="_blank" rel="noopener noreferrer">
+          <i class="info icon"></i> {{t 'Support'}}
+        </a>
+        <a class="item" href="{{if socialLinks.facebookUrl socialLinks.facebookUrl '#'}}" target="_blank" rel="noopener noreferrer">
+          <i class="facebook f icon"></i> Facebook
+        </a>
+        <a class="item" href="{{if socialLinks.twitterUrl socialLinks.twitterUrl '#'}}" target="_blank" rel="noopener noreferrer">
+          <i class="twitter icon"></i> Twitter
```

Comment:
Use translation helper here.

Reply:
@geekyd it's a translation service not a transliteration one, twitter won't translate to anything

Reply:
wait .. google translation shows.. support of proper nouns .. adding them.